src/utils/strings/prose.js

```diff
@@ -0,0 +1,28 @@
+export function formatListAsProse(list, options = {}) {
```

Comment:
__nit__: would be great to have a JSDoc for that function, especially since we have option that is `true` by default.